src/components/dialog/Dialog.js

```diff
@@ -5,38 +5,37 @@ import cx from 'classnames';
 
 import theme from './theme.css';
 
-import { Button, ButtonGroup, DialogBase, Heading3, Link } from '../../index';
+import { Button, ButtonGroup, DialogBase, Heading3 } from '../../index';
 import { IconDragMediumFilled } from '@teamleader/ui-icons';
 
 class Dialog extends PureComponent {
   bodyRef = createRef();
   dragHandleRef = createRef();
 
   getHeader = () => {
-    const { draggable, headerColor, headerIcon, onCloseClick, title } = this.props;
+    const { draggable, onCloseClick, title } = this.props;
 
-    const icon = draggable ? (
+    const dragIcon = (
       <div className={theme['drag-icon']} ref={this.dragHandleRef}>
         <IconDragMediumFilled />
       </div>
-    ) : (
-      headerIcon
     );
 
     return (
-      <DialogBase.Header color={headerColor} icon={icon} onCloseClick={onCloseClick}>
+      <DialogBase.Header icon={draggable ? dragIcon : null} onCloseClick={onCloseClick}>
         <Heading3 maxLines={1}>{title}</Heading3>
       </DialogBase.Header>
     );
   };
 
   getFooter = () => {
-    const { tertiaryAction, secondaryAction, primaryAction } = this.props;
+    const { tertiaryAction, secondaryAction, primaryAction, leftAction } = this.props;
 
     return (
-      <DialogBase.Footer>
+      <DialogBase.Footer display="flex" justifyContent={leftAction ? 'space-between' : 'flex-end'}>
+        {leftAction && <Button {...leftAction} />}
```

Comment:
PP/NTH

```suggestion
        {leftAction && <Button level="destructive" {...leftAction} />}
```

I think this will be the most common use case and you can still overwrite the level if necessary

Reply:
@lorgan3 Discussed with @stephen-tl : "it's a bit too early to decide which use case should be the default."
If we want to set a default in the future, it would not be a breaking change. So we can change this anytime later on :)